.ci/updatecli/updatecli-bump-golang.yml

```diff
@@ -1,6 +1,7 @@
 # update-cli configuration for automated go updates
 ---
 name: Bump golang-version to latest version
+pipelineid: 'updatecli-bump-golang-{{ .scm.branch }}'
```

Comment:
This should help with reusing the current PR if a bump takes longer (aka more patches in between).